src/en/users-workflows.md

```diff
@@ -0,0 +1,94 @@
+Title: Juju workflow scenarios
+TODO: 
+
+
+# Workflow scenarios
+
+To demonstrate multi-user interaction, we present several workflows. It is
+assumed that Juju is installed [link to juju install page]. This presentation
+will be based on the local cloud: LXD. Since LXD does not require credentials,
+there will be no mention of credentials below (see
+[Getting Started](./getting-started.html) for a full treatment of using LXD as
+the cloud provider).
+
+
+## Basic setup and single user
+
+**Controller creation**<br/>
```

Comment:
I feel like consistency and separating formatting from content are more important than using a manual, localized styling to compensate for a CSS fail. We should stick to the standard and continue the push to improve the standard, so that when the change comes, we don't have a ton of manual fixes to apply later.